client/src/main/java/com/arcbees/beeshop/client/application/product/ProductView.java

```diff
@@ -159,7 +163,8 @@ public void f() {
         $(addToCart).click(new Function() {
             @Override
             public void f() {
-                getUiHandlers().onAddToCartButtonClicked();
+                int itemQuantity = Integer.parseInt(quantity.getValue());
```

Comment:
Look like this is prone to error if the input `quantity` is not validated in any way?